src/pages/yoonhee/Main/CommentBox.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Comment from './Comment';
+
+class CommentBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { feedComment: '', commentList: [] };
+  }
+
+  commentInput = e => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
+
+  isKeyEnter = e => {
+    if (e.key === 'Enter') {
+      this.commentAdd();
+    }
+  };
+
+  commentAdd = () => {
+    const { feedComment, commentList } = this.state;
+    const newfeedComment = feedComment;
+    const addNewFeed = commentList.concat(newfeedComment);
+    this.setState({
+      commentList: addNewFeed,
+      feedComment: '',
+    });
```

Comment:
- `newfeedComment`라고 새로 선언하고 할당할 필요 없이, concat 메서드에 feedComment라는 state를 직접 넣어주셔도 될 것 같네요.
- concat 잘 사용해주셨는데, concat 대신에 spread operator를 사용해서 동일하게 구현해보실 수 있습니다.
연습한다고 생각하시고 찾아서 구현해보세요!
```suggestion
    const { feedComment, commentList } = this.state;
    
    this.setState({
      commentList: commentList.concat(feedComment),
      feedComment: '',
    });
```

Reply:
불필요한 변수 제거하고 말씀하신대로 spread 연산자 이용해서 만들었더니 코드가 훨씬 깔끔해져서 보기 좋아졌네용 👏
commentList: [...commentList, feedComment] !!